test/plugin/base_test.rb

```diff
@@ -338,6 +338,98 @@ def test_structured_payload_log
     end
   end
 
+  def test_autoformat_enabled_with_stackdriver_trace_id_as_trace
+    configs = [
+      APPLICATION_DEFAULT_CONFIG,
+      AUTOFORMAT_STACKDRIVER_TRACE_CONFIG
+    ]
+    configs.each do |test_config|
+      new_stub_context do
+        setup_gce_metadata_stubs
+        setup_logging_stubs do
+          d = create_driver(test_config)
+          d.emit(DEFAULT_TRACE_KEY => STACKDRIVER_TRACE_ID)
+          d.run
+          verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
+            expected_trace = \
+              "projects/#{d.instance.project_id}/traces/#{STACKDRIVER_TRACE_ID}"
```

Comment:
The interpolation happens *before* the `freeze`, so the resulting string is a constant. Plus, this would be consistent with the rest of the testing code.

Reply:
updated to be consistent with the rest of the testing code.